### PR TITLE
Use active cases

### PR DESCRIPTION
**Pull request by pieceofr**
API "confirm" & "confirm_delta" return active counts which is "confirm cases" - "recovered" - "deaths"

### CODE REVIEW

File: store/metric.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func (m *mongoDB) CollectRawMetrics(location schema.Location) (*schema.Metric, e
 				TotalPeople:             totalPeopleReport,
 				MaxScorePerPerson:       schema.TotalOfficialBehaviorWeight,
 				CustomizedBehaviorTotal: float64(behaviorToday.CustomizedCount),
-				Score:                   behaviorScore,
+				Score: behaviorScore,
```
Comment (lemonlatte) [inline store/metric.go:127]: not change this.

File: store/metric.go [hunk 2]
```diff
@@ -75,7 +75,7 @@ func (m *mongoDB) CollectRawMetrics(location schema.Location) (*schema.Metric, e
 	}
 
 	// Processing confirmed case data
-	confirmedCount, confirmDiff, confirmDiffPercent, err := m.GetCDSConfirm(location)
+	confirmedCount, confirmDiff, confirmDiffPercent, err := m.GetCDSActive(location)
```
Comment (lemonlatte) [inline store/metric.go:78]: Should this variable name be renamed to `activeCount`, `activeDiff` and `activeDiffPercent`, respectively.

**Comment by nodestory** [comment]
Reminder: update Jupyter notebook.

File: store/metric.go (hunk 1 above)
Comment (pieceofr) [reply store/metric.go:127]: ok

File: store/metric.go (hunk 2 above)
Comment (pieceofr) [reply store/metric.go:78]: yes

**Pull request merged**